addons/point_of_sale/static/src/js/models.js

```diff
@@ -1631,6 +1797,17 @@ exports.Product = Backbone.Model.extend({
         // pricelist that have base == 'pricelist'.
         return price;
     },
+
+    get_display_price: function(pricelist, quantity) {
+        if (this.pos.config.iface_tax_included === 'total') {
+            var taxes = [];
+            this.taxes_id.forEach(id => taxes.push(this.pos.taxes_by_id[id]));
+            var all_taxes = this.pos.compute_all(taxes, this.get_price(pricelist, quantity), 1, this.pos.currency.rounding);
+            return all_taxes.total_included;
```

Comment:
```suggestion
            const taxes = this.taxes_id.map(id => this.pos.taxes_by_id[id]);
            const allPrices = this.pos.compute_all(taxes, this.get_price(pricelist, quantity), 1, this.pos.currency.rounding);
            return allPrices.total_included;
```
It's better to use `map` to calculated `taxes`. Also, I'd rename `all_taxes` to `allPrices`.